cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithCordapp.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         corda_group = 'net.corda'
-        corda_release_version = '3.0-SNAPSHOT' // TODO: Set to 3.0.0 when Corda 3 is released
+        corda_release_version = '3.1-corda'
```

Comment:
Do these Gradle tests actually work yet? These `build.gradle` files are test resources, and so we can expand properties via a `processTestResources {}` block.